File: src/js/usuarios.js
export class Usuarios extends Personas {
    user;
    clave;
    empresa;
    constructor(user, clave, empresa) {

    }

    get getUser() {
        return user;
    }

    set setUser(newUser) {
        this.user = newUser;
    }

    get getClave() {
        return clave;
    }

    set setClave(newClave) {
        this.clave = newClave;
    }

    get getEmpresa() {
        return empresa;
    }

    set setEmpresa(newEmpresa) {
        this.empresa = newEmpresa;
    }
}

File: Usuarios.java
public class Usuarios extends Personas {
    private String user;
    private int clave;
    private String empresa;

    // Getter
    public String getUser() {
        return user;
    }

    public int getClave() {
        return clave;
    }

    public String getEmpresa() {
        return empresa;
    }

    // Setter
    public void setUser(String newUser) {
        this.user = newUser;
    }

    public void setClave(int newClave) {
        this.clave = newClave;
    }

    public void setEmpresa(String newEmpresa) {
        this.empresa = newEmpresa;
    }
}
